### ISSUE TITLE
Codelists needed

### ISSUE THREAD

**Issue by matthewberesford92** [opensafely/vte-diagnoses-and-deaths#4]
Codelists needed:
@joshuamutio @Jongmassey - will work on pulling these lists together. Where possible, will find ones on opencodelists and X-ref with list I have from a colleague. 

**Primary diagnosis**
- [X] VTE - ICD
- [X] VTE - snomed

**Secondary diagnoses** 
*snomed for TPP
@joshuamutio - what do you need to interrogate HES? ICD10?
- [ ]Cancer
- [ ] Metastatic cancer
- [ ] 'Acute infection' excluding covid
- [ ] COVID
- [ ] Raised BMI >30
- [ ] Ischaemic heart disease
- [ ] Heart failure
- [ ] 'Respiratory disease'
- [ ] 'Connective tissue disease'
- [ ] Pregnancy
- [ ] Thrombophilia

**Medication**
- [X] Tranexamic acid
- [ ] Combined hormonal contraception
- [ ] Tamoxifen
- [ ] Raloxifene

**Comment by matthewberesford92** [comment]
Hi @joshuamutio 

As discussed today I have now created a new list for oestrogen-containing meds exlcuding topical preparations which can be found here (https://www.opencodelists.org/codelist/user/matthewberesford92/oestrogen-based-drugs-excluding-topical-preparations/0a2b9272/)

**Comment by matthewberesford92** [comment]
@joshuamutio 

do you need an equivalent ICD-10 codelist for all of the medical diagnoses we are looking for (e.g. ischaemic heart disease, chronic resp disease etc) so you can search HES as well as TPP?

**Comment by speed-vm** [comment]
@matthewberesford92 @Jongmassey @joshuamutio I'm back from parental leave just in case you need my assistance. Jon / Josh I work with the national clinical leads for VTE prevention in my other job, and can support from within that role as needed.

**Comment by Jongmassey** [comment]
That's ace - I'd forgotten that this was your area of expertise and your offer of help is very welcome

**Comment by matthewberesford92** [comment]
Thanks @speed-vm this is great news

**Comment by speed-vm** [comment]
@Jongmassey @matthewberesford92 How are you getting on? 

Matt, perhaps we could just grab 20-30 minutes in next couple of weeks. (email didn't work the other day when I was trying to check in)

**Comment by matthewberesford92** [comment]
HI @speed-vm 
It would be good to check in. We have a made a bit of progress but not loads. One issue has been Josh's access (or lack of) to the results server which has made iterating our analytic code more difficult. Hopefully we have sorted this now as Josh has dug out his old (non M1 chip) laptop. 
We have our weekly check in on Thursday's at 12....I am on nights this weekend but do you want to join next week (23rd November)?

**Comment by speed-vm** [comment]
Yes please, Thursday 23rd at 12 works for me. Looking forward to catching up.

Please could you forward over the invite?

Hope nights ok!


BW

Vicky

________________________________

**Comment by matthewberesford92** [comment]
Sure, what is your email again vicky? I lost your address as my nhs.net email was blocked.

Thanks
Matt

**Comment by speed-vm** [comment]
***@***.***

Thank you!


Vicky Speed

Specialist Pharmacist - Anticoagulation

Kings College Hospital - Princess Royal University Hospital

Internal - Pager: 737*KH3704 External via switchboard 01689 863 000

***@***.******@***.***>



________________________________
